### PR TITLE
Add new migration guides

### PR DESCRIPTION
**Pull request by jessebraham**

### CODE REVIEW

**Review by MabezDev — APPROVED** [review]
Thanks!

**Comment by bugadani** [comment]
I'd suggest holding off until we know we don't have to do a point release.

**Review by SergioGasquez — APPROVED** [review]
Ill leave the approval so it can be merged once we have some feedback from the release

**Pull request merged**
